desktop/src/main/fileProcessing/workers/copyWorker.ts

```diff
@@ -182,6 +182,14 @@ const copyDirectoryInBatches = async (
 
     totalFileCount = await countFiles(source);
 
+    if (totalFileCount === 0) {
+      // Folder is empty
+      parentPort?.postMessage({
+        type: "emptyFolder",
+        path: source,
+      });
```

Comment:
_Empty folder * message emitted_ twice
This msg is duplicated to `countFiles.ts`

Reply:
See above